src/pages/ProfilePage.tsx

```diff
@@ -21,133 +20,113 @@ import PressableWithoutFocus from '@components/Pressable/PressableWithoutFocus';
-    const timezone = lodashGet(details, 'timezone', {});
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const avatar = details?.avatar || UserUtils.getDefaultAvatar();
+    const fallbackIcon = details?.fallbackIcon ?? '';
+    const login = details?.login ?? '';
+    const timezone = details?.timezone;
 
     // If we have a reportID param this means that we
     // arrived here via the ParticipantsPage and should be allowed to navigate back to it
-    const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyAccountIDs([accountID]) && !_.isEmpty(timezone);
-    let pronouns = lodashGet(details, 'pronouns', '');
-    if (pronouns && pronouns.startsWith(CONST.PRONOUNS.PREFIX)) {
+    const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyAccountIDs([accountID]) && !isEmptyObject(timezone);
+    let pronouns = details?.pronouns ?? '';
+    if (pronouns?.startsWith(CONST.PRONOUNS.PREFIX)) {
         const localeKey = pronouns.replace(CONST.PRONOUNS.PREFIX, '');
-        pronouns = props.translate(`pronouns.${localeKey}`);
+        pronouns = translate(`pronouns.${localeKey}` as TranslationPaths);
     }
 
     const isSMSLogin = Str.isSMSLogin(login);
     const phoneNumber = getPhoneNumber(details);
     const phoneOrEmail = isSMSLogin ? getPhoneNumber(details) : login;
 
-    const isCurrentUser = props.session.accountID === accountID;
-    const hasMinimumDetails = !_.isEmpty(details.avatar);
-    const isLoading = lodashGet(details, 'isLoading', false) || _.isEmpty(details);
+    const isCurrentUser = session?.accountID === accountID;
+    const hasMinimumDetails = !isEmptyObject(details.avatar);
+    const isLoading = Boolean(details?.isLoading) || isEmptyObject(details);
 
     // If the API returns an error for some reason there won't be any details and isLoading will get set to false, so we want to show a blocking screen
     const shouldShowBlockingView = !hasMinimumDetails && !isLoading;
 
-    const statusEmojiCode = lodashGet(details, 'status.emojiCode', '');
-    const statusText = lodashGet(details, 'status.text', '');
+    const statusEmojiCode = details?.status?.emojiCode ?? '';
+    const statusText = details?.status?.text ?? '';
     const hasStatus = !!statusEmojiCode;
     const statusContent = `${statusEmojiCode}  ${statusText}`;
 
-    const navigateBackTo = lodashGet(props.route, 'params.backTo');
+    const navigateBackTo = route?.params?.backTo;
 
-    const shouldShowNotificationPreference = !_.isEmpty(props.report) && props.report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN;
-    const notificationPreference = shouldShowNotificationPreference ? props.translate(`notificationPreferencesPage.notificationPreferences.${props.report.notificationPreference}`) : '';
+    const shouldShowNotificationPreference = !isEmptyObject(report) && report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN;
```

Comment:
`report` could be null/undefined. Even though we have checked `!isEmptyObject(report)` I think we still should update to `report?.notificationPreference` for safer

Reply:
We will read `report.notificationPreference` only in the case when the object is not empty

`!isEmptyObject(report)` will guarantee that we are dealing with an object

Reply:
Yepp this is NAB